src/bitcoin-cli.cpp

```diff
@@ -190,6 +190,15 @@ int CommandLineRPC(int argc, char *argv[])
                         throw CConnectionFailed("server in warmup");
                     strPrint = "error: " + error.write();
```

Comment:
This line `strPrint = "error: " + error.write();` get's overwritten at L198 when `error.isObject()` (currently always).
Hmm... i someone relies on a/the output of a RPC client application like bitcoin-cli he should definitively rethink the API design and probably directly use the JSON RPC API.